<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1193</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1193</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Resume Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>891</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Personal Info</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editFirstName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>19</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Middle Name (optional)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editMiddleName">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>39</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>581</x>
       <y>19</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editLastName">
     <property name="geometry">
      <rect>
       <x>581</x>
       <y>39</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E-Mail Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editEmailAddress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPhoneNumber">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Birth Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>581</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nationality</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editNationality">
     <property name="geometry">
      <rect>
       <x>581</x>
       <y>90</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editAddress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="editBirthDate">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>2</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile Photo (optional)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editProfilePhoto">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>140</y>
       <width>267</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFindProfilePhoto">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>139</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>187</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Work Experience</string>
    </property>
    <widget class="QTableView" name="tableWorkExperience">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>561</width>
       <height>271</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddWorkExperienceEntry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Entry</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveWorkExperienceEntry">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>22</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Entry</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>187</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Education</string>
    </property>
    <widget class="QTableView" name="tableEducation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>561</width>
       <height>271</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddEducationEntry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Entry</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveEducationEntry">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>21</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Entry</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionChoose_Template"/>
    <addaction name="action_Export_As_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChoose_Template">
   <property name="text">
    <string>Choose &amp;Template</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="action_Export_As_PDF">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Export As PDF</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
